runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -484,4 +530,51 @@ public void onFocusChanged(FocusChanged focusChanged)
 			setHotKeyPressed(false);
 		}
 	}
+
+	private void notifyValuableItem(GroundItem item, ValueThreshold threshold)
+	{
+		final Player local = client.getLocalPlayer();
+		final StringBuilder notificationStringBuilder = new StringBuilder()
+			.append("[")
+			.append(local.getName())
+			.append("] received ");
+
+		if (threshold == ValueThreshold.INSANE)
+		{
+			notificationStringBuilder.append("an ");
+		}
+		else
+		{
+			notificationStringBuilder.append("a ");
+		}
+
+		notificationStringBuilder.append(threshold);
+
+		if (threshold != ValueThreshold.HIGHLIGHTED)
+		{
+			notificationStringBuilder.append(" value");
+		}
+
+		notificationStringBuilder.append(" drop: ")
+			.append(item.getName());
+
+		if (item.getQuantity() > 1)
+		{
+			notificationStringBuilder.append(" x ").append(item.getQuantity());
+
+			if (item.getQuantity() >= MAX_QUANTITY)
```

Comment:
Abstracting this logic to its own method would also allow for the removal of the `if else` construct by utilizing returns